app/src/main/resources/edu/wpi/first/shuffleboard/app/PlaybackControls.fxml

```diff
@@ -8,44 +8,55 @@
       spacing="8">
-    <Button fx:id="recordButton" onAction="#toggleRecord"/>
-    <HBox fx:id="playbackControls" alignment="CENTER">
+    <HBox>
         <padding>
-            <Insets topRightBottomLeft="2"/>
+            <Insets topRightBottomLeft="5"/>
         </padding>
-        <Button onAction="#previousFrame">
-            <graphic>
-                <ImageView>
-                    <Image url="/edu/wpi/first/shuffleboard/app/icons/icons8-Rewind-16.png"/>
-                </ImageView>
-            </graphic>
-        </Button>
-        <Button fx:id="playPauseButton" onAction="#togglePlayPause"/>
-        <Button onAction="#stopPlayback">
-            <graphic>
-                <ImageView>
-                    <Image url="/edu/wpi/first/shuffleboard/app/icons/icons8-Stop-16.png"/>
-                </ImageView>
-            </graphic>
-        </Button>
-        <Button onAction="#nextFrame">
-            <graphic>
-                <ImageView>
-                    <Image url="/edu/wpi/first/shuffleboard/app/icons/icons8-Fast Forward-16.png"/>
-                </ImageView>
-            </graphic>
-        </Button>
-        <HBox alignment="CENTER" spacing="4">
-            <Scrubber fx:id="progressScrubber"/>
-            <ToggleSwitch text="Loop" fx:id="loopingSwitch" alignment="CENTER" textAlignment="CENTER"/>
-            <Label fx:id="progressLabel"/>
+        <Label fx:id="connectionIndicator">Disconnected!</Label>
+    </HBox>
+    <Region HBox.hgrow="ALWAYS"/>
+    <HBox>
+        <Button fx:id="recordButton" onAction="#toggleRecord"/>
+        <HBox fx:id="playbackControls" alignment="CENTER">
+            <padding>
+                <Insets topRightBottomLeft="2"/>
+            </padding>
+            <Button onAction="#previousFrame">
+                <graphic>
+                    <ImageView>
+                        <Image url="/edu/wpi/first/shuffleboard/app/icons/icons8-Rewind-16.png"/>
+                    </ImageView>
+                </graphic>
+            </Button>
+            <Button fx:id="playPauseButton" onAction="#togglePlayPause"/>
+            <Button onAction="#stopPlayback">
+                <graphic>
+                    <ImageView>
+                        <Image url="/edu/wpi/first/shuffleboard/app/icons/icons8-Stop-16.png"/>
+                    </ImageView>
+                </graphic>
+            </Button>
+            <Button onAction="#nextFrame">
+                <graphic>
+                    <ImageView>
+                        <Image url="/edu/wpi/first/shuffleboard/app/icons/icons8-Fast Forward-16.png"/>
+                    </ImageView>
+                </graphic>
+            </Button>
+            <HBox alignment="CENTER" spacing="4">
+                <Scrubber fx:id="progressScrubber"/>
+                <ToggleSwitch text="Loop" fx:id="loopingSwitch" alignment="CENTER" textAlignment="CENTER"/>
+                <Label fx:id="progressLabel"/>
+            </HBox>
```

Comment:
This needs to be in a separate component, putting it as part of the PlaybackIndicator doesn't make any sense architecturally.